frontend/src/scenes/plugins/PluginDrawer.tsx

```diff
@@ -47,114 +49,135 @@ export function PluginDrawer(): JSX.Element {
     }, [editingPlugin?.name])
 
     return (
-        <Drawer
-            forceRender={true}
-            visible={!!editingPlugin}
-            onClose={() => editPlugin(null)}
-            width="min(90vw, 420px)"
-            title={editingPlugin?.name}
-            footer={
-                <>
-                    <div style={{ display: 'flex' }}>
-                        <div style={{ flexGrow: 1 }}>
-                            {canDelete && (
-                                <Popconfirm
-                                    placement="topLeft"
-                                    title="Are you sure you wish to uninstall this plugin?"
-                                    onConfirm={editingPlugin ? () => uninstallPlugin(editingPlugin.name) : () => {}}
-                                    okText="Yes"
-                                    cancelText="No"
-                                >
-                                    <Button style={{ color: 'var(--red)', float: 'left' }} type="link">
-                                        <DeleteOutlined /> Uninstall
-                                    </Button>
-                                </Popconfirm>
-                            )}
-                        </div>
-                        <div>
-                            <Button onClick={() => editPlugin(null)} style={{ marginRight: 16 }}>
-                                Cancel
-                            </Button>
-                            <Button type="primary" loading={loading} onClick={form.submit}>
-                                Save
-                            </Button>
-                        </div>
-                    </div>
-                </>
-            }
-        >
-            <Form form={form} layout="vertical" name="basic" onFinish={savePluginConfig}>
-                {editingPlugin ? (
-                    <div>
-                        <div style={{ display: 'flex', marginBottom: 16 }}>
+        <>
+            <Drawer
+                forceRender={true}
+                visible={!!editingPlugin}
+                onClose={() => editPlugin(null)}
+                width="min(90vw, 420px)"
+                title={editingPlugin?.name}
+                footer={
+                    <>
+                        <div style={{ display: 'flex' }}>
+                            <div style={{ flexGrow: 1 }}>
+                                {canDelete && (
+                                    <Popconfirm
+                                        placement="topLeft"
+                                        title="Are you sure you wish to uninstall this plugin?"
+                                        onConfirm={editingPlugin ? () => uninstallPlugin(editingPlugin.name) : () => {}}
+                                        okText="Yes"
```

Comment:
More "conscious" UX would be including the verb here, i.e. "Yes, uninstall" (conversely "No, cancel").